pandas/core/frame.py

```diff
@@ -6471,6 +6471,11 @@ def dropna(
         Series.dropna : Drop missing values.
         Index.dropna : Drop missing indices.
 
+        Notes
+        --------
+        Setting dropna to true will elimate rows containing any missing data 
+        from the pivot_table.
```

Comment:
This doesn't seem to make a lot of sense. This method has nothing to do with pivot_table, and dropna is not a parameter, can't be set to anything. Maybe you may want to close this PR, try to understand better what you want to do, and start again?